core/src/tpu.rs

```diff
@@ -239,6 +241,23 @@ impl Tpu {
         )
         .unwrap();
 
+        // Streamer for Alpenglow
+        let SpawnServerResult {
+            endpoints: _,
+            thread: alpenglow_quic_t,
+            key_updater: alpenglow_stream_key_updater,
+        } = spawn_server_multi(
+            "solQuicAlpenglow",
```

Comment:
The thread name only allows 15 characters in linux. we need to shorten more, how about "solQuicAlpglw"?

Reply:
Done.